src/plays/motivate-me/components/footer/index.js

```diff
@@ -0,0 +1,8 @@
+import './index.css'
+export default function Footer () {
+    return(
+        <div className="footer">
+          Developed by <a href="https://github.com/Hat52" target="_blank" rel="noreferrer" data-replace="Hamza Ali"><span>Hamza Ali</span></a>
+        </div>
+    )
+}
```

Comment:
@Hat52 , We don't need this footer, we already show the creator info in play headers.